### ISSUE TITLE
New re-work = New name

### ISSUE THREAD

**Issue by bartolomesoriano** [mistervee/skin.madnox#11]
_From @MadMikeDoc on May 23, 2017 8:24_

@mistervee @bartolomesoriano @meowmoo 

So, been thinking about this and as it seems this is going to be a complete ground up re-work of the entire skin from images to code so what are your thoughts of a rename to turn it into a brand new skin?

The roots of the skin will always be mentioned in the credits etc. but with the work going on by the team I wondered if perhaps we should just go the full break and create a new skin using Madnox as basis?

_Copied from original issue: mistervee/skin.madnox.backup#17_

**Comment by bartolomesoriano** [comment]
_From @mistervee on May 23, 2017 9:14_

Sounds good.

The only thing i can think of it will always have that aeon style to it and some of the textures used will be from the original aeon (hoping to replace all textures from aeon nox 4 though).

The question remains is what to call it? 
Something similar but different?
Due to its origin and wanting to avoid using the name aeon my thoughts would be 'xeon' or 'zeon'. With my preference to the former.

**Comment by bartolomesoriano** [comment]
_From @meowmoo on May 23, 2017 9:20_

I dont really care about the Name, as long as it look and feel like Madnox :)

'xeon' reminds me on Intel :P

**Comment by bartolomesoriano** [comment]
Sure, let's go for it! Like @meowmoo, I really like the MadNox feel and options.  I am all for names that show transformation, revival, so how about triumph? One word that reflects achievement which generally comes from hard work. Or how about MadEvo? It's an evolution of the great work already done by Mike and the original team. Thoughts?

@MadMikeDoc @mistervee @meowmoo

**Comment by bartolomesoriano** [comment]
_From @meowmoo on May 24, 2017 1:11_

noea xon :dizzy_face:

**Comment by bartolomesoriano** [comment]
@MadMikeDoc , @meowmoo , @mistervee I have been thinking about this some more, and one of my concerns about changing the name is that if we do, then we wouldn't be able to continue using the logo schimi created for the skin, which I love! I'm all for dropping the Aeon from the name, and like Mike said, we will keep the roots of the skin mentioned always in the credits.

Thoughts?

**Comment by bartolomesoriano** [comment]
_From @mistervee on May 25, 2017 23:36_

The logo is great and Madnox is a great name.

**Comment by bartolomesoriano** [comment]
_From @meowmoo on May 26, 2017 21:45_

Than keep it!

I like it too

**Comment by bartolomesoriano** [comment]
_From @mistervee on June 1, 2017 23:40_

@MadMikeDoc @meowmoo @bartolomesoriano 

At this point there is not really much that can be cleaned up in the skin. Mostly re-work remains. I would like to also setup transifex auto translation commits.

I am thinking we need to rebase the skin. Meaning removing all commit history so we can break ties with aeon nox 4. The easiest way will involve making a new repo and re-uploading in its current state. All the issues and wiki will need to be copy and pasted over.

EDIT: There may be another way to do this. We can Unfork the repo and keep commit history between certain dates. It is a bit more complex to do and i will need to experiment a bit on a different github account. Really just need a starting date/commit for madnox. The other option is to contact github and put in a help request.

**Comment by bartolomesoriano** [comment]
@mistervee, let's go for the rebase.  I guess we can add the comments in the commits that we don't want to lose to issues. If you want I can either create issues where we don't have them or copy the comments to existing issues. Just let me know which way you want to go

**Comment by bartolomesoriano** [comment]
@mistervee, @MadMikeDoc, @meowmoo, I was thinking that since having all the different views and options is what makes MadNox great, I could go through all the views and document which media type we are currently using it for, and what options are offered with each view type.  Perhaps in a spreadsheet that we can then reference to decide what to work on, what stays and what doesn't, and also we can create at some point a more comprehensive MadNox wiki that users can reference.  Thoughts?

**Comment by bartolomesoriano** [comment]
_From @mistervee on June 2, 2017 3:51_

@bartolomesoriano 
Not so sure about the user wiki. The intention was to make it a coders wiki to keep a consistent look.

For the views it sounds like a good idea, i don't think it is required at this time. The focus is getting the skin in a clean state with very few options and adding them back in the future. It will be a up to the person re-coding the view what they want to include at the time. An open issue should state who is working on what xml or a new one should be opened for group discussion with ideas put forward.

It might seem like a bit of a mess at the moment with the direction. My sole focus is to clean up code and re-establish a clear direction.

@all
I have spent some time trying to figure out keeping the past commits and to unfork the repo. Both are not an issue. The main issue is with size of the skin on github. If you pull from the repo as a fresh skin its 750+mb. Downloading a zip file is 55mb. The 750mb comes form all of the history of texture deletions and additions. Moving forward this is not ideal, this really needs to be cleared out for a fresh start.

My thoughts moving forward;
If we want a fresh start and a brand new skin using madnox as a basis i would say we need to slim down as much as we can and re-upload the skin github. Removing all the history from the the day it is done and anything we need to add back we can refer to the old code for a copy and paste.

**Comment by bartolomesoriano** [comment]
_From @meowmoo on June 2, 2017 6:2_

Sounds like  a plan, let's go for it

**Comment by bartolomesoriano** [comment]
_From @MadMikeDoc on June 2, 2017 6:50_

@mistervee 

The rebase sounds ok, I am already in the mindset that I'll have to go back to Krypton code to pull in 'Madnox' features to views etc once the re-work is complete as it is basics only to begin with for view re-work, although it is hard to 'throw away' so much of my work but if it helps the skin then ok.

Transifex sounds good as well if it helps with strings.

Good luck with the rebase and creating a clean 'Madnox' to continue the re-work on.

**Comment by bartolomesoriano** [comment]
_From @mistervee on June 2, 2017 7:27_

@MadMikeDoc @meowmoo @bartolomesoriano 
As we are all in agreement i will start in about 12 hours after a goods nights sleep.

I would not look at it throwing away work. We have copy and paste. Most importantly knowledge.

If anyone has work they need to push to repo now is time.

**Comment by bartolomesoriano** [comment]
_From @mistervee on June 2, 2017 7:37_

Regarding the open issues, they can be moved using this tool
https://github-issue-mover.appspot.com/
